packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -531,6 +531,6 @@ export class ProjectActions {
     }
 
     // @ts-ignore - because of the conditional above, we know that devServer isn't a function
-    return WIZARD_FRAMEWORKS.find((framework) => framework.configFramework === config?.component?.devServer.framework)
+    return WIZARD_FRAMEWORKS.find((framework) => framework.configFramework === config?.component?.devServer?.framework)
```

Comment:
Rather than add the optional here, I think it's better to tweak the guard above:

```ts
if (!config?.component?.devServer || typeof config?.component?.devServer === 'function') {
  return undefined
}
...
```

Reply:
I like this suggestion 👍🏻